simtools/util/general.py

```diff
@@ -770,3 +776,93 @@ def get_log_excerpt(log_file, n_last_lines=30):
         f"{collect_final_lines(log_file, n_last_lines)}\n\n"
         "====================================================================\n"
     )
+
+
+def convert_2D_to_radial_distr(hist2d, xaxis, yaxis, bins=50, max_dist=1000):
```

Comment:
Bit surprised to see this in `general.py`. It seems to be me a very specific function, and not needed in most of the code. Need to discuss what to do.

Reply:
This is a generic mathematical conversion (from hist2d to hist1d), I think it might be useful at some point for other 2d distributions